e2e/testvalues/values.go

```diff
@@ -91,3 +91,8 @@ var LocalhostClientFeatureReleases = semverutil.FeatureReleases{
 		"v7.1",
 	},
 }
+
+// AllowAllClientsWildcardFeatureReleases represents the releases the allow all clients wildcard was released in.
+var AllowAllClientsWildcardFeatureReleases = semverutil.FeatureReleases{
+	MajorVersion: "v9",
```

Comment:
Couldn't we add this feature in v8.1?

Reply:
I think we could!

Reply:
I left `v9` as the major version and added `v8.1` in the minor versions.

Also added the backport label

Reply:
Let me update the docs before merging, because we probably have a few places where we should now mention that you don't need to update the allowed clients if you add a new client.